modules/updating-eus-to-eus-upgrade-console.adoc

```diff
@@ -0,0 +1,49 @@
+// Module included in the following assemblies:
+//
+// * updating/preparing-eus-eus-upgrade.adoc
+
+:_content-type: PROCEDURE
+[id="updating-eus-to-eus-upgrade-console_{context}"]
+= EUS-to-EUS upgrade using the web console
+
+.Prerequisites
+
+* Have access to the web console as a user with `admin` privileges.
+
+.Procedure
+
+. Upgrade any OLM Operators to versions that are compatible with both versions you are updating to. You can find these steps on the link:https://docs.openshift.com/container-platform/4.11/operators/admin/olm-upgrading-operators.html#olm-changing-update-channel_olm-upgrading-operators[Preparing for an Operator update] page.
+
+. Verify that all machine config pools display a status of `UPDATED` and that no machine config pool displays a status of `UPDATING`.
++
+To view the status of all machine config pools, click *Compute* -> *MachineConfigPools* and review the contents of the *Update status* column.
++
+[NOTE]
+====
+If your machine config pools have an `Updating` status, please wait for this status to change to `Updated`. This process could take several minutes.
+====
+
+. Set your channel to `eus-4.12`. 
++
+To set your channel, click *Administration* -> *Cluster settings* -> *Channel*. You can edit your channel by clicking on the current hyperlinked channel.
+
+. Pause the machine config pools that you want to skip reboots on. You can perform this action on the *MachineConfigPools* tab under the *Compute* page. Select the vertical ellipses next to the machine config pool you'd like to pause and click *Pause updates*.
```

Comment:
Same concern with the cli doc. Could we mention that pause worker machine pool directly?

Reply:
Absolutely - does this change the procedure at all? Or is this simply a wording change?

Reply:
i.e.: "Pause the worker machine pools that you want to skip reboots on." ?

Reply:
good question. basically, we're not only skipping reboots, but also skipping an intermediate upgrade on the workers altogether. as on whatever we're looking for pausing _all_ the workers, or _only_ the workers we're interested in upgrading this particular way, depends on the answer to [my previous comment](https://github.com/openshift/openshift-docs/pull/56680#discussion_r1127696847) on which hopefully Scott would be able to clarify.

Reply:
Gotcha - I'll tag him as well :) @sdodson what do you think?

Reply:
or @wking ?

Reply:
Updated based on the previous comment :) we are pausing all machine config pools except for the master pool